edgelet/contrib/debian/control

```diff
@@ -2,8 +2,7 @@ Source: iotedge
 Section: admin
 Priority: extra
 Maintainer: Azure IoT Edge Devs
-Build-Depends: debhelper (>=9),
-               dh-systemd
+Build-Depends: debhelper (>=10.2.5)
```

Comment:
I think >=10 is fine here.  It gives us a bit more flexibility on the requirements, but it's new enough not to require "dh-systemd"

Reply:
Yep, shall be fixing it.  3 very similar PRs makes it pretty confusing for everyone. So I haven't been pushing changes to all 3 at the same time. Do intend to round this off and in the 3rd PR too.